includes/parserfunctions/CargoDeclare.php

```diff
@@ -366,7 +366,7 @@ public static function run( Parser $parser ) {
 			$viewTableMsg = wfMessage( 'cargo-cargotables-viewtablelink' )->parse();
 			$text .= " [[$pageName|$viewTableMsg]].";
 		} else {
-			$text .= "\n\n[".$parser->getTitle()->getFullURL( [ 'action' => 'recreatedata' ] )." ".wfMessage( 'cargo-createdatatable' )->parse()."]";
+			$text .= "\n\n[" . $parser->getTitle()->getFullURL( [ 'action' => 'recreatedata' ] ) . " " . wfMessage( 'cargo-createdatatable' )->parse() . "]";
```

Comment:
Intended?

Reply:
Yeah, it was our line, and it would trip CS checks, so I decided to just make it conform to style guides